draft-barnes-acme.md

```diff
@@ -1527,30 +1551,19 @@ It MUST NOT contain any characters outside the URL-safe Base64 alphabet.
 }
 ~~~~~~~~~~
 
-A client responds to this challenge by signing a JWS object and provisioning it
-as a resource on the HTTP server for the domain in question.  The payload of the
-JWS MUST be a JSON dictionary containing the fields "type", "token", and
-"tls" from the ACME challenge and response (see below), and no other fields.  If
-the "tls" field is not included in the response, then validation object MUST
-have its "tls" field set to "true".  The JWS MUST be signed with the client's
-account key pair. This JWS is NOT REQUIRED to have a "nonce" header parameter
-(as with the JWS objects that carry ACME request objects), but MUST otherwise
-meet the guidelines laid out in {{terminology}}.
+A client responds to this challenge by creating an authorized keys object for
+this challenge and the client's account key and provisioning it as a resource on
+the HTTP server for the domain in question.
```

Comment:
Goes back to previous comment. Shouldn't that provisioned resource be JWS signed as well?